src/Mantasflowers.Domain/Entities/ProductInfo.cs

```diff
@@ -0,0 +1,11 @@
+namespace Mantasflowers.Domain.Entities
+{
+    public class ProductInfo : BaseEntity
+    {
+        public decimal Price { get; set; }
+
+        public int LeftInStock { get; set; } // TODO: still unknown where we will get this from
+
+        public int? DiscountPercent { get; set; }
```

Comment:
DiscountPercent could be better of as a decimal to avoid dependence on division by 100.